solutions/classes/flight.js

```diff
@@ -0,0 +1,29 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+	idx++;
+	return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+const Flight = class{
+	constructor(upTime,downTime){
+		this.upTime=upTime;
+		this.downTime=downTime;
+	}
+	calculateFlight(){
+		var upTimeArr=this.upTime.split(':');
+		upTimeArr=upTimeArr.map((x)=>parseInt(x));
```

Comment:
Please don't use `map`. Use explicit variables. e.g. `upTimeHrs`, `upTimeMins` 
Always remember that our students are beginners. We want to make the code as simple as possible even if it means long and non-elegant code.

Reply:
@sailesh-ramayanam  `map` is removed, and used variables `upTimeHrs` and `upTimeMins`.